docs/platform/object-storage/how-to-use-object-storage-acls-and-bucket-policies/index.md

```diff
@@ -32,6 +32,10 @@ In this guide you will learn:
 
 - You'll also need the [*canonical ID*](#retrieve-a-user-s-canonical-id) of every user you wish to grant additional permissions to.
 
+{{< note >}}
+Currently, you can create a new canonical ID, only by creating a completely new Linode account. A canonical ID is not assigned to a limited access user on an existing Linode account.
```

Comment:
The 'only' is in a weird position here. The sentence should read "Currently, you can _only_ create a new canonical ID by creating..."

Reply:
fixed as suggested.